apps/numpy_dlpack/dlpack/to_numpy.py

```diff
@@ -0,0 +1,77 @@
+import ctypes
+import numpy as np
+from .dlpack import _c_str_dltensor, DLManagedTensor, DLTensor
+
+ctypes.pythonapi.PyCapsule_IsValid.restype = ctypes.c_int
+ctypes.pythonapi.PyCapsule_IsValid.argtypes = [ctypes.py_object, ctypes.c_char_p]
+
+ctypes.pythonapi.PyCapsule_GetPointer.restype = ctypes.c_void_p
+ctypes.pythonapi.PyCapsule_GetPointer.argtypes = [ctypes.py_object, ctypes.c_char_p]
+
+def array_interface_from_dl_tensor(dlt):
```

Comment:
Shall we prefix this with underscore?

```suggestion
def _array_interface_from_dl_tensor(dlt):
```

Reply:
Renamed: both the definition and the invocation.